src/scss/cs/competition.scss

```diff
@@ -13,3 +13,123 @@
     font-size: 19px;
   }
 }
```

Comment:
Can we make the background of this main section be `#FFF` please? the contrast is off because of the `.content-body` setting the background to some weird opacity. see:

Current:
<img width="986" alt="image" src="https://github.com/user-attachments/assets/2c9ee26b-1895-4040-9870-aa3b7d0b8f6c">

Desired:
<img width="918" alt="image" src="https://github.com/user-attachments/assets/0c1536f7-0081-48d6-8573-c32b56a0ca1a">